src/encoded/static/components/item-pages/components/FilesInSetTable.js

```diff
@@ -3,7 +3,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import url from 'url';
-import { console, object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { console, object, analytics } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['analytics' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)